src/components/UserInfo/UserInfo.tsx

```diff
@@ -1 +1,19 @@
-export const UserInfo = () => {};
+import React from 'react';
+import { User } from '../../types/User';
+
+interface Props {
+  user: User,
+}
+
+export const UserInfo: React.FC<Props> = ({ user }) => {
+  const {
+    name,
+    email,
+  } = user;
+
+  return (
+    <a className="UserInfo" href={`mailto:${email}`}>
```

Comment:
Consider adding `rel='noopener noreferrer'` to the anchor tag for better security. 🔒